test/boulder-config.json

```diff
@@ -149,7 +149,7 @@
   },
 
   "pa": {
-    "dbConnect": "mysql+tcp://policy@localhost:3306/boulder_policy_integration",
+    "dbConnect": "mysql+tcp://policy@localhost:3306/boulder_policy_integration?readTimeout=800ms&writeTimeout=800ms",
```

Comment:
I went back and forth on this. We already do some URI construction to ensure that we have `parseTime=true`. I agree it would be cleaner to specify the MySQL timeouts in boulder-config.json, but that would require a much more invasive refactor, changing the signature of NewDbMap, which is used everywhere. Filed an issue: https://github.com/letsencrypt/boulder/issues/1505